Source/WebKit/NetworkProcess/Classifier/WebResourceLoadStatisticsStore.h

```diff
@@ -111,6 +115,8 @@ class WebResourceLoadStatisticsStore final : public ThreadSafeRefCountedAndCanMa
     using StorageAccessPromptWasShown = WebCore::StorageAccessPromptWasShown;
     using StorageAccessScope = WebCore::StorageAccessScope;
     using RequestStorageAccessResult = WebCore::RequestStorageAccessResult;
+    using Domain = WebCore::RegistrableDomain;
```

Comment:
This is a bit weird. I think we could remove this and just use `RegistrableDomain` in the `setLoginStatus` function declaration.

Reply:
I removed it, but I don't remember if I added this line

Reply:
apparently removing it causes error in this:
```
void NetworkConnectionToWebProcess::setLoginStatus(...) {
resourceLoadStatistics->setLoginStatus(WTFMove(**domain**), loggedInStatus, WTFMove(completionHandler));}
```

Reply:
Hmm I wouldn't expect removing this to cause a compiler error there. Make sure you also update the `setLoginStatus` function declaration below in this file to use `RegistrableDomain&&` instead of `Domain&&`.